falcon_multiqc/commands/query.py

```diff
@@ -0,0 +1,204 @@
+import click
+import subprocess
+import sys
+from database.crud import session_scope
+from database.models import *
+from sqlalchemy import Float
+from sqlalchemy.orm import load_only,Load
+from database.process_query import create_new_multiqc, create_csv
+
+# example command:
+# falcon_multiqc query --tool_metric --multiqc -d path
+# User will get prompted due to -tm flag, command will ask for which tool_metrics to filter on
+# examples
+# verifybamid AVG_DP < 28
+# picard_insertSize MEAN_INSERT_SIZE > 565
+# picard_wgsmetrics PCT_EXC_DUPE <= 0.04
+# Then hit 'enter'
+
+@click.command()
+@click.option('--select', is_flag=True, required=False, help = "Enter what you want to select on, e.g. select for sample_name, batch, etc. (deafult is sample_name)")
```

Comment:
Hey :) I think the click.options need a double check. Most are marked as flags, when they are not flags. They need their correct type added. Whether that's click.STRING or path or whatever, see types here: https://click.palletsprojects.com/en/7.x/parameters/

Reply:
I'm still not sure what type it should be lol, like for the meantime I didn't want any input to come after specifying the option because any input would contain spaces (like picard_wgsmetrics PCT_EXC_DUPE <= 0.04, or sample_name batch), so by making them flags it seemed to do what I wanted. Is there another way of doing that? So the you don't have to put stuff after the option?